bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js

```diff
@@ -152,12 +159,28 @@ class FeatureDataPluginUIHandler extends StateHandler {
             newState.sorting = newActiveLayerFeatures && newActiveLayerFeatures.length ? this.determineSortingColumn(newActiveLayerFeatures) : null;
             newState.selectedFeatureIds = newActiveLayerFeatures && newActiveLayerFeatures.length ? this.getSelectedFeatureIdsByLayerId(activeLayerId) : null;
             newState.visibleColumnsSettings = this.createVisibleColumnsSettings(newActiveLayerFeatures);
+            newState.selectByPropertiesSettings = this.createSelectByPropertiesSettings(newActiveLayerFeatures);
         }
 
         this.updateState(newState);
         this.flyoutController = showFeatureDataFlyout(this.getState(), this.getController());
     }
 
+    openSelectByPropertiesPopup () {
+        if (this.popupController) {
+            this.closePopup();
+            return;
+        }
+        this.popupController = showSelectByPropertiesPopup(this.getState(), this.getController());
+    }
+
+    closePopup () {
```

Comment:
I'm not sure if there will be other popups here, but we should rename this to match the opener-function/something like closeSelectByPropertiesPopup() to make it immediately clear what popups it is closing.

Reply:
This was on my todo list as an item to be forgotten. Fixed.